python/dask_cudf/dask_cudf/io/opt_parquet.py

```diff
@@ -124,18 +124,12 @@ def _read_partition(part, index, columns, strings_to_cats):
     if isinstance(index, list):
         columns += index
 
-    if row_groups:
-        df = cudf.io.read_parquet(
-            path,
-            row_group=row_groups[0],
-            row_group_count=len(row_groups),
-            columns=columns,
-            strings_to_cats=strings_to_cats,
-        )
-    else:
-        df = cudf.io.read_parquet(
-            path, columns=columns, strings_to_cats=strings_to_cats
-        )
+    df = cudf.io.read_parquet(
+        path,
+        row_group_list=row_groups,
```

Comment:
When there is no `_metadata` file available, `row_groups` can theoretically be passed as `None` here.  Just want to confirm that `row_group_list` can handle a `None` input?

Reply:
Yup, that's fine. None is the default value.